### DISCUSSION TITLE
Issue in Visual Studio 2022

### DISCUSSION THREAD

**Discussion by marioahc65** [community/community#16995] [Copilot Conversations]
Hello, everybody, I had an issue with Visual Studio 2022, I installed normal, and I just accept everything, it is connected, actually, it said "Github Copilot 1.14.0.1" and "Github Copilot Agent v1.14.0. 
But when I tried to generate code, GitHub does not do anything. This is so weird, I installed VS Code and it said me the issue with the SSL certificate. I installed "win-ca" in VSC  and this work perfectly but I don't know how to do it in Visual Studio 2022, if someone could help me with this, I will be too grateful. Because Github Copilot just doesn't work.

**Comment by ghost** [comment]
Got this to work?

**Reply by marioahc65** [reply]
Yes

**Reply by ghost** [reply]
> Yes

With vs 22 preview? I got vs 22 normal to work

**Reply by CyberGrand** [reply]
How - please explain?

**Reply by scott-kowalczyk-mckesson** [reply]
Try adding https://*.copilot-proxy.githubusercontent.com/ as a Trusted Site in Internet Options.
